src/Coriolis/non_traditional_beta_plane.jl

```diff
@@ -71,17 +71,17 @@ using Oceananigans.Grids: ynode, znode
 
 # This function is eventually interpolated to fcc to contribute to x_f_cross_U.
 @inline two_Ωʸw_minus_two_Ωᶻv(i, j, k, grid, coriolis, U) =
-     (  two_Ωʸ(coriolis, ynode(Center(), j, grid), znode(Center(), k, grid)) * ℑzᵃᵃᶜ(i, j, k, grid, U.w)
-      - two_Ωᶻ(coriolis, ynode(Center(), j, grid), znode(Center(), k, grid)) * ℑyᵃᶜᵃ(i, j, k, grid, U.v))
+     (  two_Ωʸ(coriolis, ynode(j, grid, Center()), znode(k, grid, Center())) * ℑzᵃᵃᶜ(i, j, k, grid, U.w)
+      - two_Ωᶻ(coriolis, ynode(j, grid, Center()), znode(k, grid, Center())) * ℑyᵃᶜᵃ(i, j, k, grid, U.v))
 
 @inline x_f_cross_U(i, j, k, grid, coriolis::NonTraditionalBetaPlane, U) =
     ℑxᶠᵃᵃ(i, j, k, grid, two_Ωʸw_minus_two_Ωᶻv, coriolis, U)
 
 @inline y_f_cross_U(i, j, k, grid, coriolis::NonTraditionalBetaPlane, U) =
-    two_Ωᶻ(coriolis,ynode(Face(), j, grid), znode(Center(), k, grid)) * ℑxyᶜᶠᵃ(i, j, k, grid, U.u)
+    two_Ωᶻ(coriolis,ynode(j, grid, Face()), znode(k, grid, Center())) * ℑxyᶜᶠᵃ(i, j, k, grid, U.u)
 
 @inline z_f_cross_U(i, j, k, grid, coriolis::NonTraditionalBetaPlane, U) =
-   -two_Ωʸ(coriolis, ynode(Center(), j, grid), znode(Face(), k, grid)) * ℑxzᶜᵃᶠ(i, j, k, grid, U.u)
+   -two_Ωʸ(coriolis, ynode(j, grid, Center()), znode(k, grid, Face())) * ℑxzᶜᵃᶠ(i, j, k, grid, U.u)
```

Comment:
this assumes a specific grid; better use the generic `ynode(i, j, k, grid,...)` ?

Reply:
Yes. All internal should use the generic. I'm even skpetical we should have the convenience versions because it lets users write non-generic code.

Reply:
fixed with 646a0fc4e